arduino/IoTkit/examples/IoTKitQuickStartExample/Setup.ino

```diff
@@ -0,0 +1,262 @@
+#include <stdio.h>
+#include <IoTkit.h>
+#include <Ethernet.h>
+#include <aJSON.h>
+#include <rgb_lcd.h>  //library from https://github.com/Seeed-Studio/Sketchbook_Starter_Kit_V2.0/tree/master/libraries/Grove_LCD_RGB_Backlight
+#include <Wire.h>
+#include <stdio.h>
+
+char  * networkSSID="CHANGE_ME";
+char  * WIFI_identity="CHANGE_ME";
```

Comment:
I prefer just char*.
You use in this file char*, char * and char  * ... as char pointer type.